app/browser-sync.ts

```diff
@@ -93,6 +95,16 @@ export class TargetLocatorSync {
     exec(this.targetLocator.window(nameOrHandle));
   }
 
+  frame(nameOrElement: ElementFinderSync | string): void {
```

Comment:
Ok, I'm with you now. In Ultra we currently select frames by name only, and I guess that got me stuck on thinking it "should" be that way.

I think I'd like to go with accepting only ElementFinderSync then to get the simplicity. But do feel free to talk me into something else if you like. :)